lib/stat_tracker.rb

```diff
@@ -59,4 +59,77 @@ def read_game_teams_csv(location_path)
       end
       game_team_data
    end
+
+   def count_of_teams
+      @data_teams.count  
+   end
+
+   def best_offense
```

Comment:
I really liked how you guys noticed the common code between all this methods and created helper methods to adhere to SRP. Great job, guys!!